### PR TITLE
Exercise 11

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

File: helpers/mongo_wrapper.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def _create_client(database_name: str):
         username = os.environ['DB_USER']
         password = os.environ['PASSWORD']
         mongo_url = os.environ['MONGO_URL']
-        return pymongo.MongoClient(f"mongodb+srv://{username}:{password}@{mongo_url}/{database_name}?retryWrites=true&w=majority")
+        return pymongo.MongoClient(f"mongodb://{username}:{password}@{mongo_url}/{database_name}?ssl=true&retryWrites=false&w=majority")
```
Comment (JackMead) [inline helpers/mongo_wrapper.py:19]: Here, you're tying the application quite tightly to the parameters required by the connection string (e.g. removing `+srv` has caused your tests to fail in Travis)
A better solution here would be to pull out this connection string as a single environment variable - your tests can continue to use e.g. the Atlas hosted DB, whilst Azure can use it's Cosmos DB

**Review by JackMead — CHANGES_REQUESTED** [review]
Looks good, I can see the app hosted in Azure and interact with it. Your Travis pipeline is currently failing though, so we should fix that up before we merge this - see my comment below

File: helpers/mongo_wrapper.py (hunk 1 above)
Comment (oskarwilliams378) [reply helpers/mongo_wrapper.py]: Fixed

**Review by JackMead — COMMENTED** [review]
Thanks for making that change, I'm happy that this is now passing in Travis, though I note your deployments aren't being picked up & we should fix that. It looks like that's an earlier change, I suspect the culprit is the `if: branch = master ` condition in your deploy lifecycle stage - we can get rid of that completely since you've already got the `on: branch: master` section.

**Comment by oskarwilliams378** [comment]
Fixed

**Pull request merged**